src/aioamazondevices/const.py

```diff
@@ -71,6 +71,7 @@
         "domain": "com.au",
         "openid.assoc_handle": f"{DEFAULT_ASSOC_HANDLE}_au",
     },
+    "pl": AMAZON_US_OVERRIDE | {"market": "https://www.amazon.com"},
```

Comment:
```suggestion
    "pl": AMAZON_US_OVERRIDE,
```

You are specifying the default as exception; is not necessary

Reply:
As I've mentioned in one of previous comments it doesn't work for me without overwriting market. There is an error that I get:
```
Selected country <PL> doesn't matches Amazon API reply:
{'input ': ['https://www.amazon.com']}
 vs 
{'amazon': 'https://www.amazon.com'}
```

Reply:
This is another issue and has nothing to do with this PR.

Reply:
I just tirggered a "update branch" so this issue is fixed.

Note: your commit message is not commitlint compliant, you need to fix it ;-)